deeplabcut/gui/tabs/train_network.py

```diff
@@ -67,13 +68,39 @@ def _on_engine_change(self, engine: Engine) -> None:
                 layout.show()
             else:
                 layout.hide()
+        self._update_snapshot_selection_widgets_visibility()
+
+    def _update_snapshot_selection_widgets_visibility(self):
+        if self.root.engine == Engine.PYTORCH:
+            self.resume_from_snapshot_label.show()
+            self.snapshot_selection_widget.show()
+        else:
+            self.resume_from_snapshot_label.hide()
+            self.snapshot_selection_widget.hide()
 
     def _set_page(self):
         self.main_layout.addWidget(_create_label_widget("Attributes", "font:bold"))
         self._generate_layout_attributes()
-        self.main_layout.addWidget(_create_label_widget(""))  # dummy label
 
-        self._pose_cfg_change(self._shuffle_display.pose_cfg)
+        self.resume_from_snapshot_label = _create_label_widget(
+            "[Optional]: Select a snapshot to resume training from", "font:bold"
+        )
+        self.resume_from_snapshot_label.setToolTip(
+            "<span style='font-weight:normal; white-space:nowrap;'>"
+            "If you've already trained a model on this shuffle, you can continue training it instead of starting "
+            "from scratch again."
+            "</span>"
```

Comment:
@n-poulsen Had to do this CSS formatting thing in order to have the ToolTip displayed in regular fond (and not bold). Doesn't look nice, I know. Apparently there could be an other approach, `QToolTip.setFont()`, but I think it affects all ToolTips in the GUI, which is not what we want...
So any opinions welcome here.

Reply:
As this works I think we can keep it as-is